modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -144,8 +144,7 @@ model Transfer {
   // has onchainTransactionId always if withdrawal
   // has OnchainTransaction relation IFF we submitted (and have tx record)
   onchainTransactionId String?
-  transactionHash      String?
-  onchainTransaction   OnchainTransaction? @relation(name: "Transaction", fields: [transactionHash], references: [transactionHash])
+  onchainTransaction   OnchainTransactionReceipt? @relation(fields: [onchainTransactionId], references: [transactionHash])
```

Comment:
change the references to be `[id]`

bc the field `onchainTransactionId` should *reference* the field `id` on the related onchainTransaction